src/main/java/tool/arenasizer/LabelWithSignedNumber.java

```diff
@@ -0,0 +1,84 @@
+package tool.arenasizer;
```

Comment:
There are a set of global components under `core.gui`, do you think this should be long there?  In case we'd need that component somewhere else?

Reply:
Well, every time when you need to show a number (or a percentage) as a label that shall be displayed with a plus or minus sign and additionally green for a positive value an red for a negative value.

Reply:
Review Finding 6: Move LabelWithSignedNumber to package 'core.gui'